src/components/dialogs/create-study-dialog/create-study-dialog.js

```diff
@@ -0,0 +1,394 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import { useForm } from 'react-hook-form';
+import { Grid } from '@mui/material';
+import { useIntl } from 'react-intl';
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import UploadNewCase from '../commons/upload-new-case';
+import {
+    createCaseWithoutDirectoryElementCreation,
+    createStudy,
+    deleteCase,
+    getCaseImportParameters,
+} from '../../../utils/rest-api';
+import {
+    HTTP_CONNECTION_FAILED_MESSAGE,
+    HTTP_UNPROCESSABLE_ENTITY_STATUS,
+} from '../../../utils/UIconstants';
+import { useSnackMessage } from '@gridsuite/commons-ui';
+import { useDispatch, useSelector } from 'react-redux';
+import ImportParametersSection from './importParametersSection';
+import { ElementType } from '../../../utils/elementType';
+import DirectorySelect from './directory-select';
+import { useNameCheck } from '../commons/use-name-check';
+import { keyGenerator } from '../../../utils/functions';
+import {
+    addUploadingElement,
+    removeUploadingElement,
+    setActiveDirectory,
+} from '../../../redux/actions';
+import { getCreateStudyDialogFormDefaultValues } from './create-study-dialog-utils';
+import {
+    API_CALL,
+    CASE_FILE,
+    CASE_UUID,
+    CURRENT_PARAMETERS,
+    DESCRIPTION,
+    FORMATTED_CASE_PARAMETERS,
+    STUDY_NAME,
+} from '../../utils/field-constants';
+import { yupResolver } from '@hookform/resolvers/yup/dist/yup';
+import yup from '../../utils/yup-config';
+import CustomMuiDialog from '../custom-mui-dialog';
+import { ErrorInput, FieldErrorAlert, TextInput } from '@gridsuite/commons-ui';
+
+const MAX_FILE_SIZE_IN_MO = 100;
+const MAX_FILE_SIZE_IN_BYTES = MAX_FILE_SIZE_IN_MO * 1024 * 1024;
+
+const CreateStudyDialog = ({ open, onClose, providedExistingCase }) => {
+    const intl = useIntl();
+    const { snackError } = useSnackMessage();
+    const dispatch = useDispatch();
+
+    const [caseFileLoading, setCaseFileLoading] = useState(false);
+
+    const activeDirectory = useSelector((state) => state.activeDirectory);
+    const selectedDirectory = useSelector((state) => state.selectedDirectory);
+    const userId = useSelector((state) => state.user.profile.sub);
+
+    const schema = yup.object().shape({
+        [STUDY_NAME]: yup.string().required(),
+        [FORMATTED_CASE_PARAMETERS]: yup.mixed(),
+        [DESCRIPTION]: yup.string().nullable(),
+        [CURRENT_PARAMETERS]: yup.mixed(),
+        [CASE_UUID]: yup.string(),
+        [CASE_FILE]: yup.mixed(),
+    });
+
+    const createStudyFormMethods = useForm({
+        mode: 'onChange',
```

Comment:
as discussed, if we want to keep the old behaviour, we will need form verifications 'onChange', otherwise, we will not need to use isCreationAllowed and we will remove this mode.. We need confirmation. thank you